extensions/wikia/ArticleVideo/scripts/article-related-video.js

```diff
@@ -0,0 +1,121 @@
+require(['wikia.window', 'wikia.tracker', 'ooyala-player'], function (window, tracker, OoyalaPlayer) {
+
+	$(function () {
+		var $video = $('#article-related-video'),
+			ooyalaVideoElementId = 'ooyala-article-related-video',
+			ooyalaVideoController,
+			track = tracker.buildTrackingFunction({
+				category: 'article-video',
+				trackingMethod: 'analytics'
+			});
+
+		function initVideo(ooyalaContainerId, videoId, onCreate) {
+			var playerParams = window.wgOoyalaParams;
+
+			ooyalaVideoController = OoyalaPlayer.initHTML5Player(ooyalaContainerId, playerParams, videoId, onCreate);
+		}
+
+		function placeRelatedVideo(player) {
+			var $articleContent = $('#mw-content-text'),
+				$articleHeaders = $articleContent.children('h2'),
+				placementCallbacks,
+				$followingSibling;
+
+			// these callbacks return candidate for following DOM element for related video
+			placementCallbacks = [
+				function () {
+					return $articleHeaders.first().nextUntil('h2', 'p, ul').last();
+				},
+				function () {
+					return $articleHeaders.eq(1).nextUntil('h2', 'p, ul').first();
+				},
+				function () {
+					return $articleHeaders.first().nextUntil('h2', 'p, ul').eq(-3);
+				}
+			];
+
+			placementCallbacks.forEach(function(func) {
+				var $followingSiblingCandidate;
+				
+				$followingSiblingCandidate = func();
+
+				if (isPlacementBetter($followingSiblingCandidate, $followingSibling)) {
+					$followingSibling = $followingSiblingCandidate;
+				}
+			});
+
+			if ($followingSibling && $followingSibling.length) {
+				$video.detach().insertBefore($followingSibling);
+
+				player.mb.subscribe(window.OO.EVENTS.PLAYBACK_READY, 'ui-title-update', function () {
+					var videoTitle = player.getTitle(),
+						videoTime = ooyalaVideoController.getFormattedDuration(player.getDuration());
+
+					$video.find('.video-title').text(videoTitle);
+					$video.find('.video-time').text(videoTime);
+
+					$video.show();
+				});
+			}
+		}
+
+		function isPlacementBetter($candidate, $currentBest) {
+			var $fakeDiv, bestWidth, candidateWidth;
```

Comment:
Shouldn't we have each variable in separate line? 
And I'd also set `$fakeDiv` value while declaring it